### DISCUSSION TITLE
How to convert primitive types to bytes or cast ptr to any type?

### DISCUSSION THREAD

**Discussion by doki23** [vlang/v#18023] [Questions and Answers]
Is there any zero copy way to get bytes like below?
```
a := 1
b := a.as_bytes() // b is a []u8
```
or maybe unsafe code
```
b := &a as []u8
```

**Comment by JalonSolov** [comment]
No, there is no "zero copy" way other than using `union`s (same as C unions).

You can use the `binary` module routines to convert _with_ copy.
